stable/selenium/templates/hub-deployment.yaml

```diff
@@ -29,15 +29,16 @@ spec:
               name: http
           livenessProbe:
             httpGet:
-              path: /grid/console
+              path: /wd/hub/status
               port: {{ .Values.hub.port }}
             initialDelaySeconds: 30
             periodSeconds: 5
             timeoutSeconds: {{ .Values.hub.livenessTimeout }}
           readinessProbe:
             httpGet:
-              path: /grid/console
+              path: /wd/hub/status
               port: {{ .Values.hub.port }}
+            initialDelaySeconds: 15
```

Comment:
since you are changing that, what do you think to make this a custom variable?

Reply:
I can do this. Would that be preferred?

Reply:
👍